scripts.js

```diff
@@ -0,0 +1,105 @@
+const recipeObject = {
+    id: 1,
+    title: "Gløgg",
+    picture_url:
+      "https://upload.wikimedia.org/wikipedia/commons/thumb/e/e2/Gl%C3%B6gg_kastrull.JPG/800px-Gl%C3%B6gg_kastrull.JPG",
+    ingredients: [
+      { NAME: "Orange zest", AMOUNT: "0.5" },
+      { NAME: "Water", AMOUNT: "200 ml" },
+      { NAME: "Sugar", AMOUNT: "275 g" },
+      { NAME: "Whole cloves", AMOUNT: "5" },
+      { NAME: "Cinnamon sticks", AMOUNT: "2" },
+      { NAME: "Spice", AMOUNT: undefined },
+      { NAME: "Bottle of red wine", AMOUNT: "1" },
+      { NAME: "Raisins", AMOUNT: "100 g" },
+      { NAME: "Slipped Almonds", AMOUNT: "50 g" },
+    ],
+    description: "Mix everything, heat it, and you are good to go!",
+  };
+
+// getting the elements from the page
+const recipeName = document.querySelector('.recipe_name')
+const recipeImg = document.querySelector('.recipeImg')
+const ingredientsList = document.querySelector('.ingredientsList')
+const cookingSteps = document.querySelector('.cookingSteps')
+
+// linking the elements to the recipe's object
+
+recipeName.innerText = recipeObject.title
+recipeImg.src = recipeObject.picture_url
+
+for (let element of recipeObject.ingredients) {
+    const listItem = document.createElement('li')
+    listItem.innerText = `${element.AMOUNT} of ${element.NAME}`;
+    ingredientsList.appendChild(listItem)
+}
+
+cookingSteps.innerText = recipeObject.description
+
+// user added recipe section
+let newRecipeObject = {
+    id: 0,
+    title: "",
+    picture_url:
```

Comment:
format the code so it's more readable.
keep the attribute and value in same lane.